Attendance/list.txt

```diff
@@ -1,19 +1,16 @@
-David Gonzalez Zumaya |25|26|27|
-Sergio Mendoza Parra |25|26|27|
-Oswaldo Vazquez Aparicio |25|26|27|
-Sebastian Herrera Bermeo |25|26|27|
-Erick Eduardo Hernández Rodríguez |25|26|27|
-Diego Flores Aleman |25|
-Braulio de Jesús Luna Ocampo |25|26|27|
-Miguel Angel Vidal Liborio |25|26|27|
-Marco Antonio Cardenas Medina |25|26|27|
+David Gonzalez Zumaya |25|
```

Comment:
Why are you removing some attendance from your partners??

Reply:
think i hace problem giving the pull, i'll check this